light-curve/tests/light_curve_py/features/test_roms.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from numpy.testing import assert_allclose
+
+from light_curve.light_curve_py import Roms
+
+
+def test_roms_1():
+    feature = Roms()
+    n = 100
+    t = np.arange(n)
+    m = np.ones(n)
+    sigma = np.ones(n)
+    actual = feature(t, m, sigma)
+    desired = 0.0
+    assert_allclose(actual, desired)
+
+
+def test_roms_2():
+    feature = Roms()
+    n = 100
+    t = np.arange(n)
+    x = np.linspace(0, 10, n)
+    m = 10 * np.sin(x)
+    sigma = np.ones(n)
+    actual = feature(t, m, sigma)
+    desired = sum(abs(m - np.median(m))) / (n - 1)
```

Comment:
It looks like a literal copy of the library code. Could you please consider to make some other test? For example, you can check what we discussed about convergence of the value for large N: sample from some random distribution and check that the value doesn't change much between N=100 and N=10000